clients/githubrepo/client.go

```diff
@@ -177,6 +177,21 @@ func (client *Client) ListStatuses(ref string) ([]clients.Status, error) {
 	return client.statuses.listStatuses(ref)
 }
 
+//ListProgrammingLanguages implements RepoClient.ListProgrammingLanguages.
+func (client *Client) ListProgrammingLanguages() (map[string]int, error) {
+	reqURL := path.Join("repos", *client.repo.Owner.Login, *client.repo.Name, "languages")
+	req, err := client.repoClient.NewRequest("GET", reqURL, nil)
+	if err != nil {
+		return nil, fmt.Errorf("request for repo languages failed with %w", err)
+	}
+	bodyJSON := map[string]int{}
+	_, errResp := client.repoClient.Do(client.ctx, req, &bodyJSON)
```

Comment:
can add a comment what the first variable returned is, and why we're ignoring it?